src/askContainer/webViews/deploySkillWebview.ts

```diff
@@ -1,44 +1,48 @@
-import * as vscode from 'vscode';
-import { AbstractWebView } from '../../runtime';
+import * as vscode from "vscode";
+import { AbstractWebView } from "../../runtime";
 
-import { ViewLoader } from '../../utils/webViews/viewLoader';
-import { deploySkill } from '../../utils/deploySkillHelper';
-import { getSkillDetailsFromWorkspace } from '../../utils/skillHelper';
-import { getOrInstantiateGitApi } from '../../utils/gitHelper';
-import { API } from '../../@types/git';
-import { Logger } from '../../logger';
-import { loggableAskError } from '../../exceptions';
-import { getSkillFolderInWs } from '../../utils/workspaceHelper';
+import { ViewLoader } from "../../utils/webViews/viewLoader";
+import { deploySkill } from "../../utils/deploySkillHelper";
+import { getSkillDetailsFromWorkspace } from "../../utils/skillHelper";
+import { getOrInstantiateGitApi } from "../../utils/gitHelper";
+import { API } from "../../@types/git";
+import { Logger } from "../../logger";
+import { loggableAskError } from "../../exceptions";
+import { getSkillFolderInWs } from "../../utils/workspaceHelper";
 
 export class DeploySkillWebview extends AbstractWebView {
     private loader: ViewLoader;
     private gitApi: API | undefined;
 
     constructor(viewTitle: string, viewId: string, context: vscode.ExtensionContext) {
         super(viewTitle, viewId, context);
-        this.loader = new ViewLoader(this.extensionContext, 'deploySkill', this);
-        getOrInstantiateGitApi(context).then(value => {
-            this.gitApi = value;
-        });
+        this.loader = new ViewLoader(this.extensionContext, "deploySkill", this);
```

Comment:
nit: Unrelated to this PR, we probably need to move these view/html file names to constants and use it.

Reply:
I can fix it for all webviews in this CR as well since it not a big change lol

Reply:
No sir, let's keep this PR scoped to what it is supposed to do.